src/components/Home/Search/Search.tsx

```diff
@@ -39,14 +38,13 @@ const Search: React.FunctionComponent = () => {
         onBlur={inputBlurHandler}
         value={searchTerm}
       />
-      {displaySearchResults && (
-        <SearchResults
-          searchResults={searchResults}
-          searchLoading={searchLoading}
-          setSearchTerm={setSearchTerm}
-          setDisplaySearchResults={setDisplaySearchResults}
-        />
-      )}
+      <SearchResults
+        searchResults={searchResults}
+        searchLoading={searchLoading}
+        setSearchTerm={setSearchTerm}
+        setDisplaySearchResults={setDisplaySearchResults}
+        displaySearchResults={displaySearchResults}
```

Comment:
seems like there are too many props provided :)
what can be improved: 
1. you can move location search request inside search results, so `searchResults ` and `searchLoading` won't be needed at all, but you'll need to app `searchTerm` prop instead
2. `setSearchTerm` and `setDisplaySearchResults` also can be removed and replaced with callback prop like `onResultSelected/resultSelectedHandler`, and in SearchResults component you can just call onResultSelected() instead of 
```
setSearchTerm('');
setDisplaySearchResults(false);
```
it will allow you to keep all logic about these two states in one place